media/utils.py

```diff
@@ -54,9 +56,17 @@ def oembed_from_id(object_id, oembed_url, service, field, pattern):
             _("Could not validate %(field)s due to invalid response from %(service)s.")
             % locals()
         ) from e
+    
+    # Youtube asset doesn't have this attribute
+    track_id = None
+    track_id_search = re.search("tracks%2F(\d+)", data.get('html', ''))
```

Comment:
usually regex patters are prefixed with `r`:

```suggestion
    track_id_search = re.search(r"tracks%2F(\d+)", data.get('html', ''))
```